stdlib/public/core/StringNormalization.swift

```diff
@@ -217,10 +217,9 @@ private func fastFill(
   // TODO: Additional fast-path: All CCC-ascending NFC_QC segments are NFC
   // TODO: Just freakin do normalization and don't bother with ICU
   var outputCount = 0
-  let outputEnd = outputBufferThreshold
   var inputCount = 0
   let inputEnd = sourceBuffer.count
-  while inputCount < inputEnd && outputCount < outputEnd {
+  while inputCount < inputEnd && outputCount < outputBufferThreshold {
```

Comment:
Same comment here: `outputEnd` is useful here because it illustrates the relationship to `inputEnd` and `outputCount` at the point of use in a way that's not obvious when using the name `outputBufferThreshold`.